server/datastore/mysql/invites.go

```diff
@@ -182,3 +185,37 @@ func (d *Datastore) loadTeamsForInvites(ctx context.Context, invites []*fleet.In
 
 	return nil
 }
+
+func (d *Datastore) UpdateInvite(ctx context.Context, id uint, i *fleet.Invite) (*fleet.Invite, error) {
+	return i, d.withRetryTxx(ctx, func(tx sqlx.ExtContext) error {
+		query, args, err := dialect.Update("invites").Where(goqu.Ex{"id": id}).Set(i).ToSQL()
+		if err != nil {
+			return ctxerr.Wrap(ctx, err, "invite update sql build")
+		}
+		_, err = d.writer.ExecContext(ctx, query, args...)
```

Comment:
```suggestion
		_, err = tx.ExecContext(ctx, query, args...)
```

Reply:
Gah! Wrapped it in a tx later on but forgot to change the actual call 🤦